src/differentials/abstract_zero.jl

```diff
@@ -34,7 +34,11 @@ struct Zero <: AbstractZero end
 
 extern(x::Zero) = false  # false is a strong 0. E.g. `false * NaN = 0.0`
 
+Base.eltype(::Zero) = Zero
```

Comment:
I think we don't need this one.
Base defines this as falling back to `eltype(typeof(x))`

```suggestion
```

Reply:
Ah, good catch